### ISSUE TITLE
Batch Normalization  Set weight problem

### ISSUE THREAD

**Issue by ouceduxzk** [fchollet/keras#2772]
Hi, I try to load weights into the BatchNormalization layer, but got an Exception: You called `set_weights(weights)` on layer "batchnormalization_1" with a  weight list of length 2, but the layer was expecting 4 weights.

Why do we have 4 weights here? I did not find out the reason in the documentation ?

Thanks

Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by NasenSpray** [comment]
Running mean, running standard deviation, gamma and beta.

**Comment by ouceduxzk** [comment]
Thanks, I was transfering the pretrained model from torch to keras, it seems in torch there isn't such gamma or beta variable, what default value should I use in general?

**Comment by NasenSpray** [comment]
Gamma and beta are the scaling and shift applied _after_ normalization (cf. BN paper). `gamma = 1, beta = 0` is "do nothing."
